src/solvers/dg_curved/containers_2d.jl

```diff
@@ -0,0 +1,90 @@
+function init_elements!(elements, mesh::CurvedMesh{2}, basis::LobattoLegendreBasis)
```

Comment:
It would be good if all functions in this file were to have at least a short comment that explains their purpose